packages/geo/src/lib/workspace/shared/wfs-workspace.ts

```diff
@@ -30,4 +30,11 @@ export class WfsWorkspace extends Workspace {
       }
     });
   }
+
+  public getLayerWksOptionNoQueryClickInTab(): boolean {
```

Comment:
Pareil ici : this.layer.options.workspace

Reply:
Fix